### ISSUE TITLE
Feature request: allow to sort field names in the diagram alphabetically

### ISSUE THREAD

**Issue by capaj** [keonik/prisma-erd-generator#122]
would be nice to have this option-especially for tables with lots of fields

**Comment by keonik** [comment]
I agree but I'm wondering if this should exist in prisma's prettier/linter or in this library. 🤔 Thoughts?

**Comment by capaj** [comment]
Ideally both would have this option, but we don't live in an ideal world 😅

**Comment by anubhabx** [comment]
Hi! I am really happy to see the repo being actively maintained again. I'd like to take this one on if it's still open.

Could you assign this to me?

**Comment by keonik** [comment]
@anubhabx appreciate the help! Take a pass at it. Let me know if you’ve got questions. Contributing guide is sparse right now so it may be some back and forth while I get that figured out.

**Comment by anubhabx** [comment]
Thanks for the assign @keonik! Before I open a PR, a few quick questions, so things stay as the repo wants it to be.

1. Should this sort only the fields inside each model, or also order the models/enums themselves? I read the issue as fields-only, but happy to do both.
2. Pure alphabetical sorting (so `id` lands wherever it sorts), or shall I keep the primary-key field pinned at the top like most ERD tools do?
3. I'm planning an opt-in generator flag (default off, so existing diagrams don't change), following the same pattern as `disableEmoji`/`ignoreEnums`. Does the name `sortFields` work for you, or would you prefer something else?

Once you let me know these, I'll put up a small PR with a test and a README note.

**Comment by keonik** [comment]
Sorry for the radio silence. Peeking at your MR now
